client/js/widgets/label.js

```diff
@@ -1,5 +1,37 @@
 import { Widget } from './widget.js';
 
+export async function unsetHTML(w) {
+  document.getElementById(w.id).innerHTML = null;
+	w.input = document.createElement('textarea');
```

Comment:
`w.input` should already be a textarea. This line shouldn't be needed.

Reply:
Deleted